src/services/index.ts

```diff
@@ -1,10 +1,13 @@
+import { GlobalDescriptorService } from './global-descriptor-service';
 import { UserDescriptorService } from './user-descriptor-service';
 
 interface IServiceMap {
+	globalDescriptorService: GlobalDescriptorService;
 	userDescriptorService: UserDescriptorService;
 }
 
 const serviceMap: IServiceMap = {
+	globalDescriptorService: new GlobalDescriptorService(),

```

Comment:
We can remove this. If you want, you can remove thsi entire file and add the same thing to userDesciprtors.